browser/ui/views/ipfs/ipfs_info_bubble_view.cc

```diff
@@ -0,0 +1,959 @@
+// Copyright (c) 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
```

Comment:
Can we use the existing code for this instead of copying the entire file? This seems like a good way to get out of sync with Chromium upgrades.
cc @bridiver I don't think we should copy this file.

Reply:
in this PR yes, but if we plan to put here the IPFS diagnostic information about the connection like on brave://ipfs maybe better to fork it though

Reply:
removed forked files